.changeset/ten-walls-collect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@bonfhir/eslint-config": minor
```

Comment:
:thinking: I wonder if this will generate a changelog entry and semver minor bump for any other Bonfhir package. Probably not is my guess.